packages/backend/src/models/UserModel.ts

```diff
@@ -138,6 +143,12 @@ export class UserModel {
                 })
                 .returning('*');
         } else {
+            if (lightdashConfig.auth.disablePasswordAuthentication) {
```

Comment:
Shouldn't this be at the service level? Since it's business logic.

I think we also need to prevent login for existing users that already have passwords. Not only new users